fitbenchmarking/example_runScripts.py

```diff
@@ -2,6 +2,8 @@
 # fit problems
 
 import os
+import sys
+sys.setrecursionlimit(10000)
```

Comment:
Please add a comment to why this is needed - i.e. a version of what you explain in the issue